Classes/Domain/Model/Video.php

```diff
@@ -49,6 +50,13 @@ class Video extends Media
      */
     protected $webM;
 
+    /**
+     * Reference to the subtitle file.
+     *
+     * @var ObjectStorage<FileReference>
+     */
+    protected $subtitles;
+
```

Comment:
You should add a constructor to avoid runtime errors. Like this:

```
    public function __construct()
    {
        $this->subtitles = new ObjectStorage();
    }
```